reference_docs/YAML_models/standoc_document_attributes.yaml

```diff
@@ -0,0 +1,724 @@
+    declaration: ":phone:, :phone{_i}:"
+
+  fax:
+    tags: ["Author info"]
+    title: "fax"
+    description: "Fax number of a person who is a contributor to the document.".
+    declaration: ":fax:, :fax{_i}:"
+
+  subdivision:
+    tags: ["Author info"]
+    title: "Subdivision of the organization"
+    description: "Subdivision of the organization that is responsible for this document".
+    added_in_version: v1.6.1
+
+  subdivision-abbr:
+    tags: ["Author info"]
+    title: "Subdivision abbreviation of the organization"
+    description: "Abbreviation of the subdivision of the organization that is responsible for this document".
+    added_in_version: v1.6.1
+
+  pub-address:
+    tags: ["Author info"]
+    title: "Address of the organization"
+    description: |
+      "Address of the organization responsible for this document, if it overrides
+      the default.
+      +
+      [NOTE]
+      --
+      Each line in a multi-line address must end with `+ \`, e.g.
+
+      [source,adoc]
+      ----
+      :pub-address: 1 Infinity Loop + \
+      California + \
+      United States of America
+      ----
+      --".
+    added_in_version: v1.6.1
+
+  pub-phone:
+    tags: ["Author info"]
+    title: "Phone number of the organization"
+    description: "Phone number of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+  pub-fax:
+    tags: ["Author info"]
+    title: "Fax number of the organization"
+    description: "Fax number of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+  pub-email:
+    tags: ["Author info"]
+    title: "Email of the organization"
+    description: "Email of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+  pub-uri:
+    tags: ["Author info"]
+    title: "URI of the organization"
+    description: "URI of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+
+  #--- Visual appearance ---#
+  body-font:
+    tags: ["Visual appearance"]
+    title: "Body font"
+    description: "Font for body text; will be inserted into CSS, overriding the default set for the particular Metanorma flavour.".
+
+  header-font:
+    tags: ["Visual appearance"]
+    title: "Header font"
+    description: "Font for headers; will be inserted into CSS, overriding the default set for the particular Metanorma flavour.".
+
+  monospace-font:
+    tags: ["Visual appearance"]
+    title: "Monospace font"
+    description: "Font for monospace; will be inserted into CSS, overriding the default set for the particular Metanorma flavour.".
+
+  htmlstylesheet:
+    tags: ["Visual appearance"]
+    title: "HTML stylesheet"
+    description: "SCSS stylesheet to use for HTML output. Defaults to built-in template for the particular Metanorma flavour. Overriding is not recommended.".
```

Comment:
Use, in fact, is not recommended. Risks as above: not following SDO branding and look-and-feel requirements; getting the HTML classes wrong; falling out of sync with any modifications we may do to the CSS (and we will, though we do try to avoid it)